app/assets/javascripts/voabarxmlfrontend.js

```diff
@@ -140,10 +140,11 @@ $(document).ready(function() {
         e.preventDefault();
         var councilTaxUploadForm = this;
 
-        function submitError(error){
+        function submitError(error, jqXHR){
```

Comment:
What does `jqXHR` stands for?

Reply:
It's part of jquery ajax API. https://api.jquery.com/Types/#jqXHR